sgx/ias-types/src/parsed.rs

```diff
@@ -0,0 +1,392 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+//! Parsed IAS JSON contents
```

Comment:
This is the equivalent of `ias::verify::VerificationReportData`, which is meant to be the parsed contents of the JSON string contained in the `report::Report` structure in this crate. I'm open to bikeshedding the name of this thing, as I'm not super pleased with `ParsedReport` (as I wasn't with the predecessor `VerificationReportData`.